### DISCUSSION TITLE
How to set partial text readonly but the rest writable in TextEdit?

### DISCUSSION THREAD

**Discussion by iamazy** [slint-ui/slint#6764] [Q&A]
Just like the following image:

<img width="567" alt="image" src="https://github.com/user-attachments/assets/2e6594cb-553b-4f0e-83be-966c991ca03d">

**Comment by ogoffart** [comment]
Similar to https://github.com/slint-ui/slint/discussions/6748, it is possible with a Text and a TextInput next to eachother.

**Reply by TheColorRed** [reply]
Or two TextInput's where one is in "read-only" mode.
